src/sentry/monitors/models.py

```diff
@@ -74,14 +74,24 @@ class MonitorEnvironmentValidationFailed(Exception):
     pass
 
 
+class MonitorObjectStatus(ObjectStatus):
```

Comment:
Let's not extend ObjectStatus. I think I removed a lot of these, just duplicate the values in `ObjectStatus` instead